includes/class-wc-siftscience-options.php

```diff
@@ -21,25 +21,24 @@ class WC_SiftScience_Options {
 		private const SCHEMA = 'siftsci_';
 
 		public const GUID    = self::SCHEMA . 'guid';
-		public const API_KEY = self::SCHEMA . 'api_key';
-		public const JS_KEY  = self::SCHEMA . 'js_key';
 		public const STATS   = self::SCHEMA . 'stats';
+		public const JS_KEY  = self::SCHEMA . 'js_key';
+		public const API_KEY = self::SCHEMA . 'api_key';
 
 		public const SEND_STATS  = self::SCHEMA . 'send_stats';
 		public const NAME_PREFIX = self::SCHEMA . 'name_prefix';
 		public const STATS_API   = 'https://sift.fermiac.staat.us';
 
-		public const THRESHOLD_GOOD    = self::SCHEMA . 'threshold_good';
+		public const IS_API_SETUP      = self::SCHEMA . 'is_api_setup';
 		public const THRESHOLD_BAD     = self::SCHEMA . 'threshold_bad';
-		public const AUTO_SEND_ENABLED = self::SCHEMA . 'auto_send_enabled';
-		public const MIN_ORDER_VALUE   = self::SCHEMA . 'min_order_value';
 		public const LOG_LEVEL_KEY     = self::SCHEMA . 'log_level';
-		public const IS_API_SETUP      = self::SCHEMA . 'is_api_setup';
+		public const THRESHOLD_GOOD    = self::SCHEMA . 'threshold_good';
```

Comment:
What is the reason for re-ordering these lines? For example, it now seems strange to me that THRESHOLD_BAD     and THRESHOLD_GOOD   are separated.

Reply:
longer line comes l8r

Reply:
I think that's ok sometimes, but in this case better to keep things logically grouped/nearby.